### DISCUSSION TITLE
Chinese Digital Calipers 150mm

### DISCUSSION THREAD

**Discussion by Gujustud** [aderusha/DDD-Printable-Wall-Control-System#77] [General]
Call me silly, but I can't seem to figure out what to use to mount this part? My brain says it should have 2 steps coming out the sides instead?

Centerpieces/Tool_hooks/1x3 Chinese Digital Calipers 150mm.stl

Ignore, I finally figured out that 10x8 pins are needed!

**Comment by kusold** [comment]
Thanks for posting the solution, I came here with the same problem.

After learning about pins, I saw this in the README:

> Some centerpieces do not have tabs on the model to press into the sidepiece, such as the "Spacer clip-on" centerpieces used in this recipe. This is always a result of avoiding "supports" in the print job. Print out a set of these 4 pins, press-fit on either side of the "Spacer clip-on" part, then press into the left/right sidepieces along with the Locking Spacer.
